### PR TITLE
Update .gitattributes to reduce dist size

### PR DESCRIPTION
**Pull request by daniel-de-wit**
Reduced dist size from 3.9M to 72K.

https://php.watch/articles/composer-gitattributes

### CODE REVIEW

**Comment by daniel-de-wit** [comment]
@Gummibeer Can you re-open this PR?

**Comment by daniel-de-wit** [comment]
@Gummibeer Might consider reopening this?

File: .gitattributes [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-/.codeclimate.yml export-ignore
-/.gitattributes export-ignore
-/.gitbook.yaml export-ignore
-/.php_cs export-ignore
-/.scrutinizer.yml export-ignore
-/.styleci.yml export-ignore
-
-# phpunit.xml.dist and tests are not excluded so tests can be run
-# this is usefull if this package ends up one day in Debian
+/.github                   export-ignore
```
Comment (Gummibeer) [inline .gitattributes:1]: 🟰

File: .gitattributes [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
+/art                       export-ignore
```
Comment (Gummibeer) [inline .gitattributes:2]: ➕

File: .gitattributes [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
+/docs                      export-ignore
```
Comment (Gummibeer) [inline .gitattributes:3]: 🟰

File: .gitattributes [hunk 4]
```diff
@@ -1,11 +1,10 @@
...
+/tests                     export-ignore
```
Comment (Gummibeer) [inline .gitattributes:4]: ➕

File: .gitattributes [hunk 5]
```diff
@@ -1,11 +1,10 @@
...
+/.gitattributes            export-ignore
```
Comment (Gummibeer) [inline .gitattributes:5]: 🟰

File: .gitattributes [hunk 6]
```diff
@@ -1,11 +1,10 @@
...
+/.gitignore                export-ignore
```
Comment (Gummibeer) [inline .gitattributes:6]: ➕

File: .gitattributes [hunk 7]
```diff
@@ -1,11 +1,10 @@
...
+/.gitbook.yaml             export-ignore
```
Comment (Gummibeer) [inline .gitattributes:7]: 🟰

File: .gitattributes [hunk 8]
```diff
@@ -1,11 +1,10 @@
...
+/.php_cs                   export-ignore
+/.styleci.yml              export-ignore
```
Comment (Gummibeer) [inline .gitattributes:9]: 🟰

File: .gitattributes [hunk 9]
```diff
@@ -1,11 +1,10 @@
...
+/.phpunit.xml.dist         export-ignore
```
Comment (Gummibeer) [inline .gitattributes:10]: ➕

**Review by Gummibeer — APPROVED** [review]

**Pull request merged**